.env.example

```diff
@@ -48,9 +51,11 @@ ETHERSCAN_API_KEY=
 # These need to be deployed and set manually, ideally from circle's stablecoin-evm repo
 
 # Mainnets
+USDC=
 USDC_BASE_IMPLEMENTATION=
 USDC_OPTIMISM_IMPLEMENTATION=
 
 # Testnets
-USDC_BASE_SEPOLIA_IMPLEMENTATION=
-USDC_OPTIMISM_SEPOLIA_IMPLEMENTATION=
\ No newline at end of file
+TESTNET_USDC=
```

Comment:
Why not USDC_ETHEREUM_TESTNET_IMPLEMENTATION?

Reply:
on it

Reply:
Ok, I think we could avoid the blockchain name if it is ETHEREUM